ENIGMAsystem/SHELL/Graphics_Systems/OpenGL3/OPENGL3Std.cpp

```diff
@@ -37,6 +37,9 @@ extern string shader_get_name(int i);
 } // namespace enigma_user
 
 namespace enigma {
+  const bool graphics_opengl_core = true;
+  const int graphics_opengl_major = 3, graphics_opengl_minor = 3;
```

Comment:
This header is supposed to be for modern GL, right? We just operate on the understanding that GL3 is the most modern GL, yes? Probably better to forward the graphics header's actual version, in this case. Honestly, I'm iffy about spoofing this for GL1.1... but really iffy about doing so for 3.x.

Reply:
* Yes
* Sort of, the most modern GL is 4.X (actually the most modern GL is Vulkan 😛)
* I don't know what you mean, the OpenGL3 system is written with the assumption that GL 3.3 is supported.
* I'm still not sure I understand you. How is it a spoof? The OpenGL1 system is supposed to fallback to GL1.1 and vertex arrays if it can't do vbos. The OpenGL3 system doesn't call functions newer than GL3.3.

Reply:
Ok: 43d780d4e0a161e2f45a9c9222a52fbc3d1c102b

I am just letting this be your call. To do like you want, we just omit the version number altogether in the attributes and only tell it core or compatibility. I just hope everybody understands well enough not to allow feature creep without doing fallbacks.

Reply:
![image](https://user-images.githubusercontent.com/3212801/56858832-c9789800-694e-11e9-9989-7de7baa5644b.png)

Reply:
Ok: 2efc2ea990392458899e1f47ae83ae7fe9480844

I decided to move Harri's debug stuff out of the Win32-GL3 bridge so that it applies to all OpenGL bridges on all platforms. Now, that required that I also set the debug bit for SDL, which worked just fine. However, I then noticed SDL is not giving us a core profile even though we explicitly request one. It gives me a 4.5 compatibility profile, so I can not get the above desired debug error message. This is in fact a **_bug_** in SDL.

```cpp
  SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK,SDL_GL_CONTEXT_PROFILE_CORE);
```

It will work and create a core profile if I specify version minor and major. However, that is not necessary with WGL, and should not be with SDL either. Again, this is an SDL bug.